timescale-container/crate-exporter.py

```diff
@@ -69,21 +69,13 @@ class CrateSql:
 
     @staticmethod
     def to_string(x) -> str:
-        """
-        Convert the input into a Crate string.
-        E.g. input   ~~~> 'input'
-        """
-        v = str(x)  # TODO consider escaping?
-        return f"'{v}'"
+
+        return f"'{x}'"
```

Comment:
how is this going to help? :-)

```python
>>> x = "howzit'!'"
>>> f"'{x}'"
"'howzit'!''"
```

whereas the single quotes in `x` should've been escaped: `'howzit\'!\''`. Or something similar, can't remember offhand what the escape sequence is for Crate...